src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -180,8 +181,15 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
         return
 
     # TODO(cathy): add check for OrganizationOption for comment bot
-    if not features.has("organizations:pr-comment-bot", organization):
-        logger.error("github.pr_comment.feature_flag_missing", extra={"organization_id": org_id})
+    if not (
```

Comment:
Does this PR mean we can take out the TODO on line 183?

Reply:
yes